openmc/material.py

```diff
@@ -560,6 +560,73 @@ def add_element(self, element, percent, percent_type='ao', enrichment=None):
         for nuclide in element.expand(percent, percent_type, enrichment):
             self.add_nuclide(*nuclide)
 
+    def add_material(self, material, percent, percent_type='ao'):
+        """Mix a material to the material
```

Comment:
Maybe "Mix a material into this material"? It's a little unclear as is. Also, I would recommend having a paragraph (after the one-liner but before Parameters) that describes this in a little more detail. In particular, how is this mixing performed? Are the nuclides from the other material added in (increasing the density of the current material), or does everything scale so that the density is preserved? Right now, a user won't be able to figure out without reading the source code.